nestkernel/nest_names.h

```diff
@@ -418,6 +418,7 @@ extern const Name sdev;             //!< Used in pulsepacket_generator
 extern const Name send_buffer_size; //!< mpi-related
 extern const Name senders;          //!< Recorder parameter
 extern const Name shift_now_spikes; //!< Used by spike_generator
+extern const Name sigma;       //!< Inverse steepness of erfc_neuron
```

Comment:
The indentation is wrong here. Please fix it to make Travis (clang-format) happy!